### DISCUSSION TITLE
Generate prefix.schema.table but prefix is not part of the schema name

### DISCUSSION THREAD

**Discussion by UddipakBhuyan** [sqlalchemy/sqlalchemy#11606] [Usage Questions] [answered]
Is it possible for select query to generate as below
```
select * from prefix.schema.table
# or 
select * from "prefix"."schema"."table"
```
In above example `prefix` is not part of the schema and should be separate. I have tried initialising `Table(..., schema="prefix.schema")` but this generates below query which doesn't work for my use case.

```
select * from "prefix.schema"."table"
```

**Comment by UddipakBhuyan** [comment, marked as answer]
I have found the solution to this issue over [here](https://github.com/sqlalchemy/sqlalchemy/discussions/10102#discussioncomment-6439123). Closing this discussion. 

However I would like to request the 'quoted_name' solution be added in the multipart schema solution of the official documentation.

**Reply by CaselIT** [reply]
If you can spare the time, a PR would be appreciated! thanks
